tests/integration/test_model_download.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from requests import HTTPError
+
+from kagglehub import model_download
+
+HANDLE = "keras/bert/keras/bert_base_en/2"
+
+
+def test_successful_download():
+    # Test a successful model download
+    expected_path = "/usr/local/google/home/aminmohamed/.cache/kagglehub/models/keras/bert/keras/bert_base_en/2"
+    assert model_download(HANDLE) == expected_path, "Model download path did not match expected path"
+
+
+def test_model_download_invalid_handle():
+    with pytest.raises(ValueError):
+        model_download("invalid_handle")
+
+
+def test_download_specific_file():
```

Comment:
I would keep only the `test_download_multiple_files`, test case. Each integration test is slow to run, so let's try to reduce overlap.